package.json

```diff
@@ -12,6 +12,7 @@
   "dependencies": {
     "discord.js": "^12.2.0",
     "dotenv": "^8.2.0",
+    "editorconfig": "^0.15.3",
```

Comment:
Move this to `devDependencies` since it isn't directly used by the bot

Reply:
Ok